students/Lincoln_Zhang/Session4/trigrams.py

```diff
@@ -45,18 +57,27 @@ def build_trigram(words):
     return trigrams
 
 def text_gen():
```

Comment:
this is really hard to test, as it relies on random numbers -- so the results aren't consistent :-)

but are there some things you could test about it?